proto/api.proto

```diff
@@ -1137,14 +1137,14 @@ message GetPricesStreamResponse {
   TokenPrice price = 2;
 }
 
-message GetPerpPositionsRequest {
+message PerpPositionsRequest {
   Project project = 1;
   string ownerAddress = 2;
   string accountAddress = 3;
   repeated common.PerpContract contracts = 4;
 }
 
-message GetPerpPositionsResponse {
+message PerpPositionsResponse {
```

Comment:
think you misunderstood my comment here. These messages should have the `Get` in front of it. `GetPerpPosition` (field #3 here) is the one that shouldn't

Reply:
the messages that match the RPC method (e.g. `GetPerpPositions(PerpPositionsRequest)  returns (PerpPositionsResponse)`) should always have the `Get` prefix. It's jsut the data types within the `Request`/`Response` objects that should not